File: Parte 3/Get_Talk_List_By_Tags/Talk.js
const mongoose = require('mongoose');

const talk_schema = new mongoose.Schema({
    _id: String,
    speakers: String,
    title: String,
    url: String,
    duration: String,
    publishedAt: Date,
}, { collection: 'TedTok_data' });

module.exports = mongoose.model('talk', talk_schema);

File: Parte 4/TedxTok/tedxtok/lib/Models/talk.dart
class Talk{
  final String title;
  final String mainSpeaker;
  final String url;
  final String duration;

  Talk.fromJSON(Map<String, dynamic> jsonMap) :
    title = jsonMap['title'],
    mainSpeaker = (jsonMap['speakers'] ?? ""),
    url = (jsonMap['url'] ?? ""),
    duration=(jsonMap['duration']??"");
}